kaldb/src/test/java/com/slack/kaldb/chunkManager/RecoveryChunkManagerTest.java

```diff
@@ -309,6 +309,8 @@ private void testChunkManagerSearch(
     assertThat(result.snapshotsWithReplicas).isEqualTo(1);
   }
 
+  // TODO: Add a test when message ingestion fails.
+
   @Test
   public void testAddMessageWithPropertyTypeErrors() throws Exception {
```

Comment:
Same with this, as it's no longer a property type error.

Reply:
Renamed.

Reply:
Doesn't look like this was addressed

Reply:
Fixed now.